### PR TITLE
1H Swords changes based from discord feedback

### PR DESCRIPTION
**Pull request by AngryMob010**
Following got some minor speed increase edits:
Engraved Backsword
Lion Imprinted Saber
Decorated Arming Sword

also Winds Fury was completely changed. The name doesn't make it clear to people that it is a cavalry sword, resulting in confusion about its stats. so i changed it back to a normal weapon.

### CODE REVIEW

**Review by namidaka — CHANGES_REQUESTED** [review]
No. 107 Length 89 speed is a big no. I'm already adding 13% to swing speed.
This weapon is going to be awful

File: items.json [hunk 1]
```diff
@@ -15588,18 +15588,18 @@
...
         ],
-        "thrustDamage": 14,
+        "thrustDamage": 13,
         "thrustDamageType": "Pierce",
-        "thrustSpeed": 86,
+        "thrustSpeed": 87,
         "swingDamage": 30,
         "swingDamageType": "Cut",
-        "swingSpeed": 88
+        "swingSpeed": 89
```
Comment (namidaka) [inline items.json:15602]: lower either swing speed or length

File: items.json [hunk 2]
```diff
@@ -31199,19 +31199,19 @@
...
-        "thrustDamage": 24,
+        "thrustDamage": 16,
         "thrustDamageType": "Pierce",
-        "thrustSpeed": 86,
-        "swingDamage": 31,
+        "thrustSpeed": 85,
+        "swingDamage": 30,
         "swingDamageType": "Cut",
-        "swingSpeed": 76
+        "swingSpeed": 89
```
Comment (namidaka) [inline items.json:31214]: lower swing speed or length

**Review by namidaka — CHANGES_REQUESTED** [review]

**Comment by AngryMob010** [comment]
gonna cancel this. these weapons are being reworked as part of a different branch fixing lengths/speeds according to the notes made in discord

**Pull request closed without merging**